src/amp/components/Analytics.tsx

```diff
@@ -102,5 +102,50 @@ export const Analytics: React.FC<{
         </amp-analytics>`,
     ];
 
+    // Trial implementation of MoDI tracker tag. For now only to appear on the business section on AMP
+    if (section && section === 'business') {
+        scripts.push(
+            `<amp-analytics type="snowplow_v2">
+                <script type="application/json">
```

Comment:
Should we be using the `<JsonScript o={{a: b}} />` syntax?

Reply:
Not sure, I'm not familiar with that - but this is following the implementation guide given to us.